cli/server.go

```diff
@@ -1607,3 +1785,71 @@ func buildLogger(cmd *cobra.Command, cfg *codersdk.DeploymentConfig) (slog.Logge
 		}
 	}, nil
 }
+
+func connectToPostgres(ctx context.Context, logger slog.Logger, driver string, dbURL string) (*sql.DB, error) {
```

Comment:
nit: Maybe extract it to a separate file too?

Reply:
I think I'll leave it here for now since it's mostly only used for server.go and makes sense to live here to me.